sdmetrics/column_pairs/statistical/cardinality_boundary_adherence.py

```diff
@@ -0,0 +1,80 @@
+"""ColumnPair metrics based on Kullback–Leibler Divergence."""
+
+import pandas as pd
+
+from sdmetrics.column_pairs.base import ColumnPairsMetric
+from sdmetrics.goal import Goal
+
+
+class CardinalityBoundaryAdherence(ColumnPairsMetric):
+    """Cardinality Boundary Adherence metric.
+
+    Computes the percentage of synthetic parents whose cardinality
+    falls within the min/max range of cardinality in the real data.
+
+    Attributes:
+        name (str):
+            Name to use when reports about this metric are printed.
+        goal (sdmetrics.goal.Goal):
+            The goal of this metric.
+        min_value (Union[float, tuple[float]]):
+            Minimum value or values that this metric can take.
+        max_value (Union[float, tuple[float]]):
+            Maximum value or values that this metric can take.
+    """
+
+    name: 'CardinalityBoundaryAdherence'
+    goal: Goal.MAXIMIZE
+    min_value: 0.0
+    max_value: 1.0
+
+    @staticmethod
```

Comment:
@amontanez24 some metrics defines `compute_breakdown` as a `staticmethod` and other as `classmethod`. Should we be consistent? Most of the metrics use `classmethod`.

Reply:
we can make consistent. If any of them require info from the class then it should be a class method, but if none of them do then I guess it should be a staticmethod

Reply:
Yes at least the `CorrelationSimilarity`: https://github.com/sdv-dev/SDMetrics/blob/b1853d94166c04b8cbd2e22581ac269e9a6db5d8/sdmetrics/column_pairs/statistical/correlation_similarity.py#L79

Reply:
Interesting, that case I think all the methods involved could be static. It doesn't seem like it ever uses an attribute of the class

Reply:
I think it makes sense on a case by case basis. If the needs to access other methods or properties of the class, it should be a `classmethod` for convenience. Otherwise it can just be a `staticmethod` since we don't need to access anything else on the class.

Reply:
Yeah, although I think in the case of `CorrelationSimilarity` all the methods involved could be static. None of them seem to be using properties of the class